public/scripts/app.js

```diff
@@ -99,18 +109,52 @@
     // Comment Button
     $(".comment-container")
       .find(".comment-btn")
-      .on("click", function () {
+      .on("click", function (event) {
+        event.preventDefault();
         $(".comment-btn").addClass("active");
+        db.query(`INSERT INTO resource_comments (title, content, user_id, resource_id)
+         VALUES("title", "content", 5, 2);`)
+        // `, [])
+        .then(db.query(`SELECT COUNT(resource_comments.id) as count FROM resource_comments`))
+        .then((data) => {
+          let numComments = data.rows[0];
+        })
+        .then(db.query(`SELECT * FROM resource_comments WHERE resource_id = $1;`, [numComments]))
+        .then((data) => {
+          let commentInfo = data.rows[0];
+          const newComment = createCommentElement(commentInfo)
+          .then($(".comment-container").prepend("newComment"))
+          .then($(".comment-btn").removeClass("active"))
+        });
       });
+      
+
+      const createCommentElement = function(commentInfo) {
+
+        let $comment = $(
+          `<div class="comment">
+            <div class="comment-body">
+              <h4 class="comment-title">${commentInfo.title}</h4>
+              <p class="username">Posted By: ${commentInfo.username} </p>
+              <p class="comment-content">${commentInfo.description}</p>
+            </div>
+            <div>
+              <p class="comment-date">${commentInfo.updated_at}</p>
+            </div>
+          </div>`
+          );
+        return $comment;
+      };
+      
     // !!! When need to handle this with ajax, like tweeter. Once comment is rendered, the button will need to have the active class removed
 
     // Comments Load More Button
 
-    $("#load-more")
-      .find(".load-more")
-      .on("click", function () {
-        $(".load-more").addClass("active");
-      });
+    $(".load-more").find(".load-more").on("click", function() { 
+      $(this).remove();
+      $('#hidden-comments').removeClass('hidden');
+    });
+
```

Comment:
load more should probably be making a request for more comments rather than just unrevealing comments already on the page.